tests/Facebook/InstantArticles/Client/ClientTest.php

```diff
@@ -87,7 +87,7 @@ public function testRemoveArticle()
           ->expects($this->once())
           ->method('getArticleIDFromCanonicalURL')
           ->with($canonicalURL)
-          ->will($this->returnValue($articleID));;
```

Comment:
I've removed an extra `;` here, this could be done automatically using #53 and the `duplicate_semicolon` fixer, see https://github.com/FriendsOfPHP/PHP-CS-Fixer#usage:

> **duplicate_semicolon [symfony]**
> Remove duplicated semicolons.